docs/wiki/Vision.md

```diff
@@ -0,0 +1,115 @@
+# Vision
+
```

Comment:
After reading through .. I think the Vision statement page should focus on the What and the why, and then briefly touch on the "how", with linking to another page that has a more deep dive. The content is great, just a bit of organising to get the flow right.

Reply:
@ahmadabdalla  hit me up some time and let's discuss this